internal/db/schema/migrations/postgres/14/01_event.up.sql

```diff
@@ -0,0 +1,217 @@
+│ create_time     │           │                  │   
+| update_time     |           └──────────────────┘
+| version         |                     |
+└─────────────────┘                     |
+         ┼                              ┼            
+         ┼                              │            
+         │                              │            
+         │                             ╱│╲           
+         ○                     ┌─────────────────┐   
+        ╱│╲                   ╱│ event_type_enm  │   
+┌─────────────────┐     ┌──────├─────────────────┤   
+│   event_sink    │     │     ╲│                 │   
+├─────────────────┤     │      └─────────────────┘   
+│public_id        │┼────┤                            
+│config_id        │     │ 
+│event_type       |     |      
+|sink_type        |     |     ┌─────────────────────┐       
+|format_type      │     │    ╱│ event_sink_type_enm │
+└─────────────────┘     ├─────├─────────────────────┤
+         ┼              │    ╲│                     │
+         │              │     └─────────────────────┘
+         │              │     ┌───────────────────────┐    
+         ○              │    ╱│ event_format_type_enm │    
+        ╱│╲             └─────├───────────────────────┤    
+┌─────────────────┐          ╲│                       │    
+│ event_file_sink │           └───────────────────────┘    
+├─────────────────┤                                  
+│ public_id       │                                  
+│ sink_id         │                                  
+│ path            │                                  
+│ file_name       │                                  
+│ rotate_bytes    │                                  
+│ rotate_duration │                                  
+│ rotate_max_files│                                  
+└─────────────────┘                                                                                      
+*/
+
+create table event_type_enm (
+    name text primary key
+        constraint only_predefined_event_types_allowed
+        check (
+            name in (
+                'every',
+                'error',
+                'audit',
+                'observation',
+                'system'
+            )
+        )
+);
+comment on table event_type_enm is
+'event_type_enm is an enumeration table for the valid event types within '
+'the domain';
+
+create table event_sink_type_enm (
+    name text primary key
+        constraint only_predefined_event_sink_types_allowed
+        check (
+            name in (
+                'stderr',
+                'file'
+            )
+        )
+);
+comment on table event_sink_type_enm is
+'event_sink_type_enm is an enumeration table for the valid sink types within '
+'the domain';
+
+create table event_format_type_enm (
+    name text primary key
+        constraint only_predefined_event_format_types_allowed
+        check (
+            name in (
+                'json',
```

Comment:
these enums need to be re-aligned with the current valid SinkFormats: https://github.com/hashicorp/boundary/blob/ba85a3bc3fb2be769b3eaa4835eac93726c19bd1/internal/observability/event/sink_format.go#L14